fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1224,6 +1224,7 @@ public void loadImage(String imageDir) throws IOException, DdlException {
             remoteChecksum = dis.readLong();
             checksum = loadCompactionManager(dis, checksum);
             remoteChecksum = dis.readLong();
+            loadNewPrivilege(dis);
```

Comment:
You cannot put the function here if you do not write image. It will crash the image in the following case:
1. new type image is added after the newPrivilege, newLoad for example.
2. the newPrivilege is released, so usingNewPrivilege is set to true, and the newPrivilege image is written before newLoad.
3. the version that contains the newPrivilege is rollbacked. The newPrivilege image will be loaded by newLoad-loader, which is not expected.

Reply:
For now we only support new privilege on test environment. There is also no compatibility issue because the upgrade process has not implemented yet. I'll put this at the end of the` loadImage()` before 3.0 release and usingNewPrivilege is set to true.

Reply:
OK, if another new image part is added before the release of 3.0. You should remember to put this function at the end of loadImage.

Reply:
Will do. I've already added a comment on `loadPrivilege()` &&`savePrivilege()`.